tasks/sep-dev-0/002-fp-tag-sync/README.md

```diff
@@ -0,0 +1,92 @@
+# Devnet FP Upgrade - `op-contracts/v1.4.0-rc.3`
```

Comment:
Upgrade execution can be found at https://sepolia.etherscan.io/tx/0x4859f2fda193b9c76d5e96c7e586f1c4fac91e9b1eedcbe6c1ab65adf3850532

Reply:
Added this in https://github.com/ethereum-optimism/superchain-ops/pull/192/files#r1649142560